tools/ssvm-aot/ssvmr.cpp

```diff
@@ -113,10 +115,12 @@ int main(int Argc, const char *Argv[]) {
       }
     }
 
-    if (FuncType.Params.size() != Args.value().size() - 1) {
-      std::cout << "Parameter count mismatch, need " << FuncType.Params.size()
-                << ", gave " << Args.value().size() - 1 << ".\n";
-      return EXIT_FAILURE;
+    if (HasFunc) {
+      if (FuncType.Params.size() != Args.value().size() - 1) {
+        std::cout << "Parameter count mismatch, need " << FuncType.Params.size()
+                  << ", gave " << Args.value().size() - 1 << ".\n";
+        return EXIT_FAILURE;
+      }
     }
```

Comment:
If the `HasFunc` is false, should we handle the `else` case here?

Reply:
Updated workflow.